docs/commands/ft.search.md

```diff
@@ -213,6 +213,39 @@ FT.SEARCH returns an array reply, where the first element is an integer reply of
 - If a hash expires after the query process starts, the hash is counted in the total number of results, but the key name and content return as null.
 </note>
 
+### Return multiple values
+
+When the index is defined `ON JSON`, a reply for a single attribute or a single JSONPath may return multiple values when the JSONPath match multiple values, or when the JSONPath match an array. Prior to RediSearch v2.6, only the first of the matched values was returned. Starting with RediSearch v2.6, all values are returned, wrapped with a top-level array. In order to maintain backward compatibility, the default behavior with RediSearch v2.6 is to return only the first value. In order to return all the values `DIALECT 3`, or greater than `3` when available, should be used.

```

Comment:
* Please change "when the JSONPath match" to "when the JSONPath matches".
* Break down the paragraph into one sentence per line.
* Take another look at the last sentence. Should it be: "To return all the values, use `DIALECT 3` (or greater, when available)."